### PR TITLE
feat : Diff types of avatar added

### PR DESCRIPTION
**Pull request by shuklahimanshu81**
Screenshot (328)

> Added rounded avatars

> Added Square shaped avatars

> Added avatars with text.

### CODE REVIEW

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
+
+
+/* round avatar */
+
+
+
+.sm-rounded-avatar{
+    height: 40px;
```
Comment (aryan-13) [inline components/avatar/avatar.css]: Using rems is preferable over absolute units for responsiveness

File: components/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    width: 40px;
```
Comment (aryan-13) [inline components/avatar/avatar.css]: here also

File: components/avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    border-radius: var(--border-radius-f);
+}
+.md-rounded-avatar{
+    height:80px;
+    width: 80px;
+    border-radius: var(--border-radius-f);
+}
+.lg-rounded-avatar{
+    height: 120px;
```
Comment (aryan-13) [inline components/avatar/avatar.css]: Here too

**Review by aryan-13 — COMMENTED** [review]
Absolute units(px) used, which can be replaced with rems!
Else the code looks clean

File: components/avatar/avatar.css (hunk 1 above)
Comment (shuklahimanshu81) [reply components/avatar/avatar.css]: Done changes.

**Review by AnuragSahu11 — COMMENTED** [review]
Looks good to me

**Pull request merged**